tests/client/auth/test_auth_client.py

```
@@ -268,46 +269,31 @@ def path_exists_side_effect(path: str) -> bool:
         client = AuthClient()
-        client.auth_token = TraktAuthToken(
-            access_token="test_token",
-            refresh_token="test_refresh",
-            expires_in=7200,
-            created_at=int(time.time()),
-            scope="public",
-            token_type="bearer",
-        )
 
+    client.auth_token = TraktAuthToken(
+        access_token="test_token",
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to argument: "access_token"

(S106)